app/marketing/mails.py

```diff
@@ -1570,18 +1570,18 @@ def tax_report(to_emails=None, zip_paths=None, tax_year=None):
                 html, text = render_tax_report(to_email, tax_year)
                 from_email = settings.CONTACT_EMAIL
                 send_mail(
-                    from_email, 
-                    to_email, 
-                    subject, 
-                    text, 
-                    html, 
+                    from_email,
+                    to_email,
+                    subject,
+                    text,
+                    html,
                     from_name="Kevin Owocki (Gitcoin.co)",
                     categories=['marketing', func_name()],
                     zip_path=zip_paths[idx]
                 )
-            finally:    
+            finally:
                 translation.activate(cur_language)
-            
+
```

Comment:
@zoek1 try to avoid autoformatting files as it makes reviews more difficult

Reply:
Sure, i'll remove the option from my editor